ext/kv/remote.rs

```diff
@@ -312,27 +311,26 @@ fn decode_value(
   }
 }
 
-fn encode_value(value: crate::Value) -> pb::KvValue {
+fn encode_value(value: Value) -> pb::KvValue {
   match value {
-    crate::Value::V8(data) => pb::KvValue {
+    Value::V8(data) => pb::KvValue {
       data,
       encoding: pb::KvValueEncoding::VeV8 as _,
     },
-    crate::Value::Bytes(data) => pb::KvValue {
+    Value::Bytes(data) => pb::KvValue {
       data,
       encoding: pb::KvValueEncoding::VeBytes as _,
     },
-    crate::Value::U64(x) => pb::KvValue {
+    Value::U64(x) => pb::KvValue {
       data: x.to_le_bytes().to_vec(),
       encoding: pb::KvValueEncoding::VeLe64 as _,
     },
   }
 }
 
-fn encode_mutation(m: crate::KvMutation) -> pb::KvMutation {
+fn encode_mutation(m: KvMutation) -> pb::KvMutation {
   let key = m.key;
-  let expire_at_ms =
-    m.expire_at.and_then(|x| i64::try_from(x).ok()).unwrap_or(0);
+  let expire_at_ms = m.expire_at.map(|dt| dt.timestamp_millis()).unwrap_or(0);
```

Comment:
This is wrong! Durations before UNIX_EPOCH are now set to not expire, rather than expire immediately!